src/EditorFeatures/CSharpTest2/Recommendations/WhenKeywordRecommenderTests.cs

```diff
@@ -455,13 +482,43 @@ class C
     void M() { const object var = null; switch (new object()) { case var $$ break; } }
 }");
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/30848"), Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
         public async Task TestForSwitchCase_SemanticCheck_AfterClassAndLocalConstantVar_BeforeWhen() =>
     await VerifyKeywordAsync(@"
 class var { }
 class C
 {
     void M() { const object var = null; switch (new object()) { case var $$ when } }
 }");
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestForSwitchCase_SemanticCheck_AfterTypeAliasAndFieldConstantVar() =>
```

Comment:
This is not new behavior, it worked before the tests were skipped. Just adding an additional test case.